docs/advanced-usage/packaging-for-automated-evaluation.md

```diff
@@ -214,6 +214,20 @@ cases and our models.
 
 ## Appendix
 
+### Testing evaluator locally
+
+You can verify the evaluator Docker image by running it locally:
+
+```
+docker run --rm \
+  -e KOLENA_TEST_SUITE_NAME="my test suite" \
```

Comment:
nit: It's not clear from this blob that the test suite needs to be an existing test suite
```suggestion
  -e KOLENA_TEST_SUITE_NAME="${EXISTING_TEST_SUITE_NAME}" \
```